instance.tf

```diff
@@ -20,7 +20,7 @@ resource "aws_launch_configuration" "autoscaling_launch_configuration" {
   image_id                    = "${data.aws_ami.autoscaling_launch_ami.id}"
   instance_type               = "${var.instance_type}"
   key_name                    = "${var.key_pair_name}"
-  security_groups             = ["${module.autoscaling_instance_security_group.security_group_id}"]
+  security_groups             = ["${module.autoscaling_instance_security_group.security_group_id}","${var.additional_security_group_ids}"]
```

Comment:
Can we change this to this form instead?

```
security_groups             = "${concat(list(module.autoscaling_instance_security_group.security_group_id), var.additional_security_group_ids)}"
```

What I don't want to have happen - is even though there is an implied flattening of this parameter right now via list declaration, I don't want any updates to HIL and TF breaking this in a way where there's the possibility that this could turn into a [string, list] versus the [string, string, string...] that it is now.

The explicit functional declaration ensures a single flattened list is created instead.